### PR TITLE
[Remove Stream/Optional usage] LPS-176497 module: dynamic-data-mapping-form-field-type Part1

### PR DESCRIPTION
**Pull request by JanisZhang**
https://issues.liferay.com/browse/LPS-176497

previous: https://github.com/liferay-core-infra/liferay-portal/pull/3620

### CODE REVIEW

**Comment by JanisZhang** [comment]
ci:test:data-engine

**Comment by dantewang** [comment]
@JanisZhang Please see my comment and resend, or reopen if it's not an issue.

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/java/com/liferay/dynamic/data/mapping/form/field/type/internal/grid/GridDDMFormFieldValueRequestParameterRetriever.java [hunk 1]
```diff
@@ -55,11 +53,7 @@ public String get(
 		}
 
 		if (parameterValues.length == 1) {
-			jsonObject = Optional.ofNullable(
-				getJSONObject(_log, parameterValues[0])
-			).orElse(
-				jsonObject
-			);
+			jsonObject = getJSONObject(_log, parameterValues[0]);
```
[the remark this answers is not in the document]
Comment (JanisZhang) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/java/com/liferay/dynamic/data/mapping/form/field/type/internal/grid/GridDDMFormFieldValueRequestParameterRetriever.java:56]: Hi @dantewang, this method will always return `JSONFactoryUtil.createJSONObject( value )`,  if the value is NULL, this method will returns a _NULL_JSON which is "{}".

**Comment by dantewang** [comment]
ci:forward

cc @shuyangzhou @tinatian

**Pull request closed without merging**